docs/reference/settings.yaml

```diff
@@ -1494,6 +1494,14 @@ settings:
           - Default: `false`
         doc: |
           If set, enables proxying of SPDY protocol upgrades.
+      - name: "Cluster Name"
+        keys: ["name"]
+        attributes: |
+          - Config File Key: `name`
+          - Type: `string`
+          - Optional
+        doc: |
+          Runtime metrics for this policy would be available under `envoy_cluster_`*`name`* prefix.
```

Comment:
I believe you need to run the python script in the scripts directory to generate the docs.